src/components/Header/index.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { FC } from 'react';
+import AddIcon from '@mui/icons-material/Add';
+import LanguageIcon from '@mui/icons-material/Language';
+import MenuIcon from '@mui/icons-material/Menu';
+import SearchIcon from '@mui/icons-material/Search';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import logo from 'assets/img/logo.svg';
+import { NavBar } from 'components/NavBar';
+import { CustomIconButton } from 'components/shared/IconButton';
+import { IPage } from 'utils/types/navbar.type';
+
+export const Header: FC = () => {
+	const navItems: IPage[] = [
```

Comment:
move to utils/consts/navbar.ts and rename to NAVBAR. the consts should be in an uppercase